test/NCI.OCPL.Api.CTSListingPages.Tests/Tests/Models/ListingResultsComparer.cs

```diff
@@ -0,0 +1,80 @@
+using System.Linq;
+using System.Collections.Generic;
+
+namespace NCI.OCPL.Api.CTSListingPages.Tests
+{
+    /// <summary>
+    /// An IEqualityComparer for ListingResults objects.
+    /// </summary>
+    public class ListingResultsComparer : IEqualityComparer<ListingResults>
+    {
+        public bool Equals(ListingResults x, ListingResults y)
+        {
+            // If the items are both null, or if one or the other is null, return
+            // the correct response right away.
+            if (x == null && y == null)
+            {
+                return true;
+            }
+            else if (x == null || y == null)
+            {
+                return false;
+            }
+
+            bool isEqual =
+                new ResultsMetadataComparer().Equals(x.Meta, y.Meta)
+                && AreParamArraysEqual<ListingInfo, ListingInformationComparer>(x.Results, y.Results);
```

Comment:
**DO NOT CHANGE**

This is just a note, rather than something to change. [Linq provides a `SequenceEqual` method on `IEnumerable<T>`](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal?view=netcore-2.1#definition) which does the same thing as `AreParamArraysEqual`, except that it's not limited to arrays. No points off for using this method, but I find myself moving in the direction of using the one from Linq.